src/main/java/edu/kis/powp/jobs2d/drivers/adapter/LineDrawerAdapter.java

```diff
@@ -0,0 +1,43 @@
+package edu.kis.powp.jobs2d.drivers.adapter;
+
+import java.awt.Color;
+
+import edu.kis.legacy.drawer.shape.ILine;
+import edu.kis.legacy.drawer.shape.LineFactory;
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.drivers.factory.LineCustomizable;
+import edu.kis.powp.jobs2d.features.DrawerFeature;
+
+/**
+ * driver adapter to drawer with several bugs.
+ */
+public class LineDrawerAdapter implements Job2dDriver {
+    private int startX = 0, startY = 0;
+    private ILine customizableLine;
+
+    public LineDrawerAdapter(ILine line) {
+    	super();
+        customizableLine=LineFactory.getBasicLine();
+        setCustomizableLine(Color.CYAN, 5, true);
+    }
+
+    @Override
+    public void setPosition(int x, int y) {
+        this.startX = x;
+        this.startY = y;
+    }
+
+    @Override
+    public void operateTo(int x, int y) {
+        customizableLine.setStartCoordinates(this.startX, this.startY);
+        customizableLine.setEndCoordinates(x, y);
+        setPosition(x,y);
+        DrawerFeature.getDrawerController().drawLine(customizableLine);
```

Comment:
Constructor injection should be used (using a parameter). This is unnecessary direct coupling with DrawerFeature class.